dist/release.env

```diff
@@ -1,5 +1,5 @@
-VERSION=v1.1.4
-GIT_REF=974a3bf8eac67054c81b9a763bc9e162807d1e7b
-GIT_AUTHOR=Andrew Min
-GIT_KEY=DE050A451E6FAF94C677B58B9361DEC647A087BD
-GIT_TIMESTAMP=2025-05-28 10:11:33 -0400
+VERSION=v1.1.5
+GIT_REF=4ca5c0022c48debfdf5ad8a619265262f4498e54
+GIT_AUTHOR=t-vila
+GIT_KEY=
```

Comment:
This is empty because the commit this is based on is not signed by a GPG key: this is a merge commit done by github (clicking the merge button)